db_upgrade_scripts/mosip_prereg/ddl/prereg-otp_transaction.sql

```diff
@@ -1,63 +0,0 @@
--- -------------------------------------------------------------------------------------------------
```

Comment:
why is this file deleted from upgrade ?

Reply:
“prereg-otp_transaction.sql"  is NOT used in 1.1.5.5_to_1.2.0.1-B1_upgrade.sql script